client/packages/system/src/Report/components/ReportArgumentsModal.tsx

```diff
@@ -62,14 +62,14 @@ export const ReportArgumentsModal: FC<ReportArgumentsModalProps> = ({
           variant="ok"
           disabled={!!error}
           onClick={async () => {
-            onArgumentsSelected(report, data);
+            onArgumentsSelected(report, jsonData);
```

Comment:
`jsonData` is derived from `data` (adds 3 fieds), so should only get sent to the output function, not the local form state

Reply:
The three fields don't load for me anymore.

Reply:
Whereabouts? I'm seeing them in both the URL and the GraphQL query. 🤔

Reply:
![Uploading Screenshot 2024-08-01 at 5.56.40 PM.png…]()

Reply:
Ooh, I've never seen them on the modal at all, I guess I don't have a ui-schema that includes them. 🤷‍♂️

<img width="527" alt="Screenshot 2024-08-01 at 6 00 47 PM" src="https://github.com/user-attachments/assets/39f193db-1894-455a-aa71-5f12490d4767">

If they're meant to be there, I'll just need to rejig this a little bit.

Reply:
If you re-upserts any of the new reports in reports repo (still in PR) they should be there

Reply:
Okay @roxy-dao, have changed it to initialise the `data` state with these values already included. 03b1104bb